### PR TITLE
Zoom Shapes in Studio for HiDPI screens

### PR DESCRIPTION
**Pull request by Dragon-Baroque**
Commit 1 of 1 :
    gumps/VideoOptions_gump.cc
      Add resolutions up to 1600
      Pair all resolutions in 16:10 ( original 320x200 ) and 4:3
    mapedit/exult_studio.glade
      Name the GtkBox widget in the Menubar to add the Zoom widgets
    mapedit/studio.cc + h
      Parse and store the command line Zoom options
      Build and handle the Menubar Zoom widgets
      Handle keyboard + and - including Keypad to change the Zoom scale
    mapedit/objbrowse.cc, chunklst.cc, combo.cc, npclst.cc, shapelst.cc
      [ Debug ] Display event messages for the Scrollers, the Wheels
    mapedit/chunklst.cc + h, combo.cc + h
      Move the Scrollers setup when Zooming change into a new setup_info
      Switch the Scrollers to smoother - part row - than full row
    mapedit/shapedraw.cc + h
      Implement Zooming in Shape_draw::show, ExultStudio::shape_image
      Rework Shape_single which factorizes all single Shape displays
    mapedit/combo.cc, chunklst.cc, npclst.cc, shapelst.cc
      Respect the Zooming
         in the Grid of Shapes, the Selection and the vertical scolling
    mapedit/npclst.cc, shapelst.cc
      In Frames mode, respect the Zooming
         in the Grid of Shapes, the Selection and the horizontal scrolling
      Fix the vertical scrollers to handle the most bottom row

In Summary :

1. New resolutions in Exult
2. Name the MenuBar in Studio's Glade file to add the Zoom controls into it
3. studio.cc & h : provide the command line and GUI to set the Zoom
4. Chunks and Combos : Zoom the shapes, Scroll smoothly
5. NPCs and Shapes : Zoom the shapes

### CODE REVIEW

**Comment by Dragon-Baroque** [comment]
This Pull request to satisfy the Issue #278.
